core/src/main/java/org/hibernate/ogm/exception/spi/ErrorHandlingStrategy.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.exception.spi;
+
+
+/**
+ * @author Gunnar Morling
+ *
+ */
+public enum ErrorHandlingStrategy {
+	ABORT, CONTINUE;
```

Comment:
Another value could be `RETRY`, but I don't think it's needed in the first iteration. It would repeatedly invoke the failed op (and the error handler) until it either succeeds or the error handler returns ABORT/CONTINUE.